### PR TITLE
DRUP-859 #ready adds alt text field to news image thumbnail image

### PR DESCRIPTION
**Pull request by emb03**
@z3cka merging this with ucla_news change already on www-test because it has the same version on info file

### CODE REVIEW

File: www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc [hunk 1]
```diff
@@ -172,7 +172,10 @@ function uclalib_news_field_default_field_instances() {
           'fancybox__custom_crop' => 0,
           'fancybox__large' => 0,
           'fancybox__linkit_thumb' => 0,
+          'fancybox__media_thumbnail' => 0,
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc]: do not add

File: www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc [hunk 2]
```diff
@@ -172,7 +172,10 @@ function uclalib_news_field_default_field_instances() {
...
           'fancybox__medium' => 0,
+          'fancybox__staff_profile_photo' => 0,
+          'fancybox__staff_profile_photo_thumbnail' => 0,
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc]: do not add line 177 & 178

File: www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc [hunk 3]
```diff
@@ -187,7 +190,10 @@ function uclalib_news_field_default_field_instances() {
           'image_custom_crop' => 0,
           'image_large' => 0,
           'image_linkit_thumb' => 0,
+          'image_media_thumbnail' => 0,
           'image_medium' => 0,
+          'image_staff_profile_photo' => 0,
+          'image_staff_profile_photo_thumbnail' => 0,
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc]: do not add 193, 195, &196

File: www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc (hunk 1 above)
Comment (emb03) [inline www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc]: Fixed, now showing two commits with a 'squash and merge' option

File: www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc (hunk 3 above)
Comment (emb03) [inline www/sites/all/modules/features/uclalib_news/uclalib_news.features.field_instance.inc]: Fixed

**Comment by z3cka** [comment]
oops, forgot to merge... oh, well will have to cp both commits. deploying now.

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

**Pull request merged**